src/app/navigation/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -0,0 +1,40 @@
+import { CommonModule } from '@angular/common';
+import { CUSTOM_ELEMENTS_SCHEMA, Component } from '@angular/core';
+import { NavigationEnd, Router, RouterLink, RouterOutlet } from '@angular/router';
+import { unslugify } from '../../../utils';
+import { filter } from 'rxjs/operators';
+
+@Component({
+  selector: 'nav-breadcrumbs',
+  templateUrl: './breadcrumbs.component.html',
+  imports: [RouterLink, RouterOutlet, CommonModule],
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export default class BreadcrumbsNavComponent {
+  segments: Array<{ title: string; path: string }> = []; // Initialize an empty array
+
+  constructor(private router: Router) {
+    this.router.events
+      .pipe(filter((event) => event instanceof NavigationEnd))
+      .subscribe((event: any) => {
```

Comment:
not a blocker, but lets close this subscription :)

Reply:
Does this update make any sense @gokberknur ?
It is done with great help from the ChatGPT :D